src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,8 @@
+package wolox.training.repositories;
+
+import org.springframework.data.repository.Repository;
+import wolox.training.models.Book;
+
+public interface BookRepository extends Repository<Book, String> {
```

Comment:
you should extend Repository<Book, Long>, because it represents the primary key to be applied in crud methods later.

Reply:
I forgot to change this